src/DbPostgreSQLAdapter/Connection.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+#include "DbAdapterInterface/IConnection.h"
+
+namespace systelab {
+	namespace db {
+		class IDatabase;
+	}
+}
+
+namespace systelab { namespace db { namespace postgresql {
+
+	class Connection : public IConnection
+	{
+	public:
+		Connection();
```

Comment:
Default the constructor here too if you are not doing anything special on the cpp